.travis.yml

```diff
@@ -60,6 +60,6 @@ script:
   #- ./gradlew build connectedCheck
   - cd webapp
   - chmod +x gradlew
-  - ./gradlew check build && ./gradlew test
+  - ./gradlew check build # && ./gradlew test
   - npm install && npm run build
```

Comment:
Please don't change this. Make sure you have pulled --rebase before pushing.

Reply:
ok